### DISCUSSION TITLE
Output the current code page to a file on the host?

### DISCUSSION THREAD

**Discussion by emendelson** [joncampbell123/dosbox-x#4743] [Q&A]
I need to use the current code page in DOSBox-X in a script that runs on the host. Is there any way to output `chcp` to a file on the host? Or is there any other way for anything on the host to detect the current code page?

**Comment by rderooy** [comment]
You could redirect the output of a command such a `keyb` or `chcp` to a file or device (e.g. PRN) that is shared with the host. And parse that output on the host.

**Reply by emendelson** [reply]
@rderooy - Yes, that would work. Thank you.
